homeassistant/components/cover/mqtt.py

```diff
@@ -279,21 +279,26 @@ def position_message_received(topic, payload, qos):
             if self._template is not None:
                 payload = self._template.async_render_with_possible_json_value(
                     payload)
+
             if payload.isnumeric():
-                if 0 <= int(payload) <= 100:
-                    percentage_payload = int(payload)
+                int_payload = int(payload)
+                if (self._position_open == DEFAULT_POSITION_OPEN
```

Comment:
I think we should always call `find_percentage_in_range`.

Reply:
Ok.